_posts/2024-08-02-community-news-august-2024.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: single
+title: "pyOpenSci Community News: August 2024"
+excerpt: "Welcome to the first edition of pyOpenSci's Community News newsletter, where we summarize and share conversations, contributions, and news related to our diverse and vibrant community! Read on for insights into pyOpenSci sprints, Spanish translation efforts, and Hatch for Python package creation."
+author: "Jesse Mostipak"
+permalink: /blog/pyos-community-news-aug-2024.html
+header:
+  overlay_image: /images/headers/white-header.png
+  overlay_filter: rgba(20, 13, 36, 0.8)
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+
+
+## <i class="fa-regular fa-heart"></i> pyOpenSci community news
+
+As the pyOpenSci community has continued to grow, we wanted to dedicate at least one newsletter issue a month to sharing conversations, contributions, and news related to our diverse and vibrant community! This month we’re celebrating everyone who’s participated in a pyOpenSci sprint, sharing the exciting news around the Spanish translation of our Python Packaging Guide, and elevating a discussion from GitHub around the use of Hatch for Python package creation.
+
+## <i class="fa-solid fa-egg"></i> Hello, Hatch!
+
+Education is one of the [three petals of pyOpenSci](https://www.pyopensci.org/blog/pyos-education-announcement.html), and to that end we strive to create resources that help learners navigate the Python packaging ecosystem with ease. This means that when we find our community members and scientists encountering consistent issues with trying to make their workflows more open and reproducible, or trying to build a Python package, we look for ways to lower (or ideally remove!) the barrier for learners. And this was exactly the scenario we were faced with in our beginner-friendly packaging tutorials. We decided that Hatch is a great user-friendly tool that scientists can use to package and share their code. However, in developing and teaching our [Get to know Hatch tutorial](https://www.pyopensci.org/python-package-guide/tutorials/get-to-know-hatch.html), we found scientists struggling to install a tool called [`pipx`](https://github.com/pypa/pipx) that can be used to install Hatch globally, rather than in a Python environment. This was causing a host of problems (in large part because `pipx` can be tricky to install on Windows machines!)
+
+We know that it can be challenging to wrap your head around packaging, but what it boils down to is that packaging is really just a process of making something that can be installed into an environment and then reused in different places. And while we decided to use Hatch because it makes packaging life easier, it turns out that it has its own nuanced pain points that we’re still filing down! And we want to share this, because we firmly believe that even someone who is brand new to packaging can get value out of our materials.
```

Comment:
```suggestion
We know that it can be challenging to wrap your head around packaging, but what it boils down to is that packaging is really just a process of making your code installable so that it can be reused in different places. And while we decided to use Hatch because it makes packaging life easier, it turns out that Hatch has its own nuanced pain points that we’re still filing down! And we want to share this, because we firmly believe that even someone who is brand new to packaging can get value out of our materials.
```

maybe link to tutorials??